openfisca_core/columns.py

```diff
@@ -32,6 +37,7 @@ class Column(object):
     formula_class = None
     is_period_size_independent = False  # When True, value of column doesn't depend from size of period (example: age)
     is_permanent = False  # When True, value of column doesn't depend from time (example: ID, birth)
+    period_behavior = None
```

Comment:
@MattiSG: we could use some advice on naming. We are wondering how to name the "native" (bad candidate) period of a variable. For instance, `rsa` can only be called for a month, `impot_revenu` can only be called for a year.